metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -0,0 +1,156 @@
+- name: stats
+  type: group
+  description: >
+    Kibana stats and run-time metrics.
+  release: beta
+  fields:
+    - name: cluster_uuid
+      type: keyword
+      description: >
+        UUID of the Elasticsearch cluster to which Kibana connects.
+    - name: name
+      type: keyword
+      description: >
+        Kibana instance name.
+    - name: uuid
+      type: keyword
+      description: >
+        Kibana instance uuid.
+    - name: version.number
+      type: keyword
+      description: >
+        Kibana version number.
+    - name: status.overall.state
+      type: keyword
+      description: >
+        Kibana overall state.
+    - name: uptime_in_millis
+      type: long
+      description: >
+        Amount of time that the Kibana process has been running in milliseconds.
+    - name: process
+      type: group
+      description: >
+        Kibana process metrics.
+      fields:
+        - name: mem
+          type: group
+          description: >
+            Memory usage metrics of the Kibana instance.
+          fields:
+            - name: heap_max_in_bytes
+              type: long
+              description: >
+                Total amount of heap memory used by V8.
+            - name: heap_used_in_bytes
+              type: long
+              description: >
+                Amount of memory in use by V8.
+            - name: resident_set_size_in_bytes
+              type: long
+              description: >
+                The amount of space occupied in main memory for the process. Includes heap, code segment, and stack.
+            - name: external_in_bytes
+              type: long
+              description: >
+                Memory usage of C++ objects bound to JavaScript objects managed by V8.
+        - name: pid
+          type: long
+          description: >
+            Process ID of the Kibana instance.
+    - name: os
+      type: group
+      description: >
+        Operating system metrics.
+      fields:
+        - name: load_average
+          type: group
+          description: >
+            Load average metrics of the operating system.
+          fields:
+            - name: 1m
+              type: long
+              description: >
+                Average system load over the last 1 minute.
+            - name: 5m
+              type: long
+              description: >
+                Average system load over the last 5 minutes.
+            - name: 15m
+              type: long
+              description: >
+                Average system load over the last 15 minutes.
+        - name: mem
+          type: group
+          description: >
+            Memory usage metrics of the operating system.
+          fields:
+            - name: free_in_bytes
+              type: long
+              description: >
```

Comment:
For the bytes you can add `format: bytes` so the Kibana index pattern will automatically contain the unit.